charts/alfresco-search-enterprise/values.yaml

```diff
@@ -65,6 +65,12 @@ reindexing:
         # -- Key within the configmap holding the full url to connect to the
         # alfresco repository
         url: REPOSITORY_URL
+  # -- Helm hooks definition
```

Comment:
```suggestion
```

usually we document only leaf nodes to avoid having the additional entry in the docs with the entire payload as example